server/db.js

```diff
@@ -1,6 +1,11 @@
 const { Client } = require("pg");
-const client = new Client();
-await client.connect();
-const res = await client.query("SELECT $1::text as message", ["Hello world!"]);
-console.log(res.rows[0].message); // Hello world!
-await client.end();
+const fs = require("fs");
+const fastcsv = require("fast-csv");
+
+const client = new Client({
+  host: "localhost",
+  user: "postgres",
+  database: "q_a",
+  password: "password",
```

Comment:
Even though the database is not going to be consumed by the public, it's probably best that we separate sensitive information into a config file, and add it to .git ignore. Afterward, we'll need to remove this file from the repo.

Reply:
Good call! Done!